packages/aws-cdk/lib/api/util/cloudformation/stack-status.ts

```diff
@@ -46,7 +46,37 @@ export class StackStatus {
       || this.name === 'UPDATE_ROLLBACK_COMPLETE';
   }
 
+  /**
+   * Whether the stack is in a paused state due to `--no-rollback`.
+   *
+   * The possible actions here are retrying a new `--no-rollback` deployment, or initiating a rollback.
+   */
+  get rollbackChoice(): RollbackChoice {
+    switch (this.name) {
+      case 'CREATE_FAILED':
+      case 'UPDATE_FAILED':
+        return RollbackChoice.START_ROLLBACK;
+      case 'UPDATE_ROLLBACK_FAILED':
+        return RollbackChoice.CONTINUE_UPDATE_ROLLBACK;
+      case 'ROLLBACK_FAILED':
+        // Unfortunately there is no option to continue a failed rollback without
+        // a stable target state.
+        return RollbackChoice.NONE;
```

Comment:
I think this should cause an error because it doesn't achieve what the user wanted.